e2e/app/component/create-criteria-modal.ts

```diff
@@ -0,0 +1,37 @@
+import {Page} from 'puppeteer';
+import ClrIconLink from 'app/element/clr-icon-link';
+import {ElementType} from 'app/xpath-options';
+import Dialog from './dialog';
+
+const defaultXpath = '//*[contains(concat(" ", normalize-space(@class), " "), " crit-modal-container ")]';
+
+export enum PhysicalMeasurementsCriteria {
+  BMI = 'BMI',
```

Comment:
Please alphabetize these. Also, maybe document what this is for (and if it's supposed to include all the possibilities).